### PR TITLE
Removed QuestionType field in Poll models

### PR DESCRIPTION
**Pull request by jackthmp**
## Overview

QuestionType was an enum used when we had Free Response in the app. Since we now only use Multiple Choice, it's no longer needed and not represented in the upcoming API spec changes (#335). 

## Changes Made

Removed `QuestionType` and any references.

## Test Coverage

- Tested on Local Server with Alicia's `alicia/remove-fr` branch; all seems to be in the clear.

## Next Steps (delete if not applicable)

- Won't merge in until after the API spec changes are updated on the Dev/Prod server.

## Related PRs or Issues (delete if not applicable)

- #335

### CODE REVIEW

File: Pollo/Supporting/Constants.swift [hunk 1]
```diff
@@ -49,8 +49,6 @@ struct Routes {
     static let userDelete = "user/poll/delete"
     static let userDeleteLive = "user/poll/delete/live"
     static let userEnd = "user/poll/end"
-    static let userFreeResponseFilter = "user/poll/fr/filter"
-    static let userFreeResponseLive = "user/poll/fr/live"
```
Comment (jackthmp) [inline Pollo/Supporting/Constants.swift]: just random free response stuff leftover, will tackle more in #346

File: Pollo/Views/Cards/PollMiscellaneousCell.swift [hunk 2]
```diff
@@ -100,10 +100,7 @@ class PollMiscellaneousCell: UICollectionViewCell {
                 descriptionLabel.text = "Shared with group"
             }
             var unit: String
-            switch miscellaneousModel.questionType! {
-            case .multipleChoice:
-                unit = miscellaneousModel.totalVotes == 1 ? voteString : "\(voteString)s"
-            }
+            unit = miscellaneousModel.totalVotes == 1 ? voteString : "\(voteString)s"
```
Comment (jackthmp) [inline Pollo/Views/Cards/PollMiscellaneousCell.swift]: i think this is some leftover upvoting stuff?? will look into and remove

File: Pollo/Models/ListDiffableModels/Poll.swift [hunk 3]
```diff
@@ -108,19 +88,15 @@ class Poll: Codable {
     func getSelected() -> Any? {
         if userAnswers.isEmpty { return nil }
         guard let googleID = User.currentUser?.id, let answers = userAnswers[googleID], !answers.isEmpty else { return nil }
-        switch type {
-        case .multipleChoice:
-            guard let answer = answers[0].letter else { return nil }
-            return answer
-        }
+        guard let answer = answers[0].letter else { return nil }
```
Comment (kevinchan159) [inline Pollo/Models/ListDiffableModels/Poll.swift]: we can probably chain these two `guard let` and i know we check for `!answers.isEmpty` but still might be safer to do `let answer = answers.first?.letter`

File: Pollo/ViewControllers/PollBuilderViewController.swift [hunk 4]
```diff
@@ -264,53 +264,51 @@ class PollBuilderViewController: UIViewController {
...
-                        switch result {
-                        case .value:
-                            self.getDrafts()
-                        case .error(let error):
-                            print("error: ", error)
-                        }
+
+        let question = mcPollBuilder.questionText ?? ""
+        var options = mcPollBuilder.getOptions()
+       if options.isEmpty {
```
Comment (kevinchan159) [inline Pollo/ViewControllers/PollBuilderViewController.swift]: is indentation off here?

Comment (jackthmp) [reply Pollo/ViewControllers/PollBuilderViewController.swift]: Fixed!

File: Pollo/Models/ListDiffableModels/Poll.swift (hunk 3 above)
Comment (jackthmp) [reply Pollo/Models/ListDiffableModels/Poll.swift]: Yes

File: Pollo/ViewControllers/PollBuilderViewController.swift [hunk 5]
```diff
@@ -264,53 +264,51 @@ class PollBuilderViewController: UIViewController {
     // MARK: - Actions
     @objc func saveAsDraft() {
         if !canDraft { return }
-        guard let type = questionType else {
+        if !didLoad {
             print("warning! cannot save as draft before viewDidLoad")
```
Comment (chalo2000) [inline Pollo/ViewControllers/PollBuilderViewController.swift]: curious about what issue this code was fixing

**Pull request closed without merging**